docs/guides/src/main/server/configuring-database.adoc

```diff
@@ -0,0 +1,126 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+
+<@tmpl.guide
+    title="Configuring a database"
+    summary="An overview about how to configure relational databases"
+    includedOptions="db db-*">
+
+In this guide, you are going to understand how to configure the server to store data using different relational databases. You should also learn
+what are the databases supported by the server and how to proceed when using a unsupported database version.
+
+== Querying the list of supported databases
+
+The server has built-in support for different databases. You should be able to query the available databases by looking at the expected values
+for the `db` configuration option.
+
+The list of supported databases and their corresponding versions are:
+
+|===
+|Database | Version
```

Comment:
The versions are what we tests with rather than what we directly support, so would make this clear.

```suggestion
|Database | Tested Version
```